Mathlib/NumberTheory/Divisors.lean

```diff
@@ -58,6 +58,31 @@ def divisorsAntidiagonal : Finset (ℕ × ℕ) :=
   (Icc 1 n).filterMap (fun x ↦ let y := n / x; if x * y = n then some (x, y) else none)
     fun x₁ x₂ (x, y) hx₁ hx₂ ↦ by aesop
 
+--Ideally we'd want this to be defined using `List.Icc 1 n` but this doesn't exist yet
```

Comment:
You can use `List.range'` here instead: that's what it's for :)

Reply:
Good point! Done.